taxonomies/ingredients.txt

```diff
@@ -40924,6 +40959,10 @@ ciqual_food_name:en:Melon, honeydew, pulp, raw
 ciqual_food_name:fr:Melon miel ou melon honeydew, pulpe, cru
 #27 @2021-10-10
 
+<en:melon
+en:Wax gourd, inter melon
+ja:トウガン, 冬瓜
```

Comment:
```suggestion
ja:トウガン, 冬瓜
wikipedia:en:https://en.wikipedia.org/wiki/Wax_gourd
wikidata:en:Q652206
```

If you want, you can copy-paste from wikidata the translations in other languages.